Docs/reference/content/reference/bson/guidserialization/serializerchanges/serializerchanges.md

```diff
@@ -0,0 +1,17 @@
++++
+date = "2020-07-23T00:00:00Z"
+draft = false
+title = "Serializer changes"
+[menu.main]
+  parent = "GuidSerialization"
+  identifier = "GuidSerializationSerializerChanges"
+  weight = 10
+  pre = "<i class='fa'></i>"
++++
+
+# Serializer changes
+
+The following sections describe changes to the two serializers that have been changed to handle Guids differently:
```

Comment:
Perhaps just `The following sections describe changes to the two serializers to handle Guids differently`

Reply:
Done